src/state/utils.ts

```diff
@@ -1,11 +1,16 @@
 import { LOAD_MOVIES_AMOUNT } from './constants';
 
-export function getUrlParams(filter?: string, sort?: string, offset?: number) {
-  const searchParams = new URLSearchParams();
+export function getUrlParams() {
+  const searchParams = new URLSearchParams(window.location.search);
 
-  if (filter && filter !== 'all') searchParams.append('filter', filter);
-  if (sort) searchParams.append('sortBy', sort);
-  if (offset) searchParams.append('offset', offset.toString());
+  if (!searchParams.has('sortBy')) searchParams.set('sortBy', 'release_date');
```

Comment:
I would return the previous version because it is more flexible. For example, if you need to get these parameters from Redux.

Reply:
Not sure about this, because in this case I would also have to pass the parametres to other thunks like removeMovie or postMovie which use the getMovies function with this utility.